extras/config.ini

```diff
@@ -7,6 +7,7 @@ socket_buffer_size = 16384
 ;tls_cert = /etc/avalanchemq/cert.pem
 ;tls_key = /etc/avalanchemq/key.pem
 ;tls_compability_mode = false
+block_default_user_remotely = false
```

Comment:
We only block "guest" in this PR, so a default user could be another user. 
Are we calling it `block_default_user_remotely` as this might change, or should we call it `block_guest_user_remotely`?

Reply:
As I understood it, we would like to be able to change the guest user to default user. Otherwise it better be called `block_guest_user_remotely`.